### PR TITLE
New nn init copied from pytorch 1.8

### PR DESCRIPTION
**Pull request by miterion**
Currently evaluating with four seeds respectively
Seeds are:
- 329
- 835
- 1875
- 2921
- [x] FNN (`qq_su-ppo.py`)
- [x] RNN (`qq_su-ppo.py`)
- [x] Mujoco env (`hop_ppo.py`)

# Results
## FNN
<details>
<summary>Old Init</summary>

image

</details>
<details>
<summary>New Init</summary>

image

</details>

## RNN
<details>
<summary>Old Init</summary>

image

</details>
<details>
<summary>New Init</summary>

image

</details>

## Mujoco env
<details>
<summary>Old Init</summary>

image

</details>
<details>
<summary>New Init</summary>

image

</details>

### CODE REVIEW

**Comment by fdamken** [comment]
What's pyright?

**Comment by fdamken** [comment]
Can you add a comment like ”copied from torch code (2021-06-06)“?

**Comment by miterion** [comment]
> What's pyright?

Upps, that commit was not meant to be in here

File: pyproject.toml [hunk 1]
```diff
@@ -31,3 +31,6 @@ atomic = true
 multi_line_output = 3
 lines_after_imports = 2
 forced_separate = "pyrado"
+
+[tool.pyright]
+exclude = ["Pyrado/pyrado/environments/pysim/assets/*.egg"]
```
Comment (famura) [inline pyproject.toml:36]: May I ask you to remove this, since you also added it to #71

**Review by famura — CHANGES_REQUESTED** [review]
THank you very much for taking care of this. I only got a minor request.

File: pyproject.toml (hunk 1 above)
Comment (miterion) [reply pyproject.toml]: Removed through force push

**Review by famura — APPROVED** [review]
Results for the RNNs look promising

**Review by famura — APPROVED** [review]

**Pull request merged**